### ISSUE TITLE
Glue metadata ingestion unable to map type

### ISSUE THREAD

**Issue by matef** [open-metadata/OpenMetadata#1521]
I have tried to ingest two tables from Glue to OpenMetadata but I got an error saying "unable to map type 'double' to metadata schema" however I can see the Double data type in the permitted enum but in the upper case. The same issue happened with the Decimal type.
 
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/metadata/ingestion/source/glue.py", line 130, in ingest_tables
    table_entity = Table(
  File "pydantic/main.py", line 360, in pydantic.main.BaseModel.__init__
  File "pydantic/main.py", line 986, in pydantic.main.validate_model
  File "pydantic/fields.py", line 590, in pydantic.fields.ModelField.validate
  File "pydantic/fields.py", line 619, in pydantic.fields.ModelField._validate_sequence_like
  File "/usr/local/lib/python3.9/site-packages/metadata/ingestion/source/glue.py", line 97, in get_columns
    yield Column(
  File "pydantic/main.py", line 362, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for Column
dataType
  value is not a valid enumeration member; permitted: 'NUMBER', 'TINYINT', 'SMALLINT', 'INT', 'BIGINT', 'BYTEINT', 'FLOAT', 'DOUBLE', 'DECIMAL', 'NUMERIC', 'TIMESTAMP', 'TIME', 'DATE', 'DATETIME', 'INTERVAL', 'STRING', 'MEDIUMTEXT', 'TEXT', 'CHAR', 'VARCHAR', 'BOOLEAN', 'BINARY', 'VARBINARY', 'ARRAY', 'BLOB', 'LONGBLOB', 'MEDIUMBLOB', 'MAP', 'STRUCT', 'UNION', 'SET', 'GEOGRAPHY', 'ENUM', 'JSON' (type=type_error.enum; enum_values=[<DataType.NUMBER: 'NUMBER'>, <DataType.TINYINT: 'TINYINT'>, <DataType.SMALLINT: 'SMALLINT'>, <DataType.INT: 'INT'>, <DataType.BIGINT: 'BIGINT'>, <DataType.BYTEINT: 'BYTEINT'>, <DataType.FLOAT: 'FLOAT'>, <DataType.DOUBLE: 'DOUBLE'>, <DataType.DECIMAL: 'DECIMAL'>, <DataType.NUMERIC: 'NUMERIC'>, <DataType.TIMESTAMP: 'TIMESTAMP'>, <DataType.TIME: 'TIME'>, <DataType.DATE: 'DATE'>, <DataType.DATETIME: 'DATETIME'>, <DataType.INTERVAL: 'INTERVAL'>, <DataType.STRING: 'STRING'>, <DataType.MEDIUMTEXT: 'MEDIUMTEXT'>, <DataType.TEXT: 'TEXT'>, <DataType.CHAR: 'CHAR'>, <DataType.VARCHAR: 'VARCHAR'>, <DataType.BOOLEAN: 'BOOLEAN'>, <DataType.BINARY: 'BINARY'>, <DataType.VARBINARY: 'VARBINARY'>, <DataType.ARRAY: 'ARRAY'>, <DataType.BLOB: 'BLOB'>, <DataType.LONGBLOB: 'LONGBLOB'>, <DataType.MEDIUMBLOB: 'MEDIUMBLOB'>, <DataType.MAP: 'MAP'>, <DataType.STRUCT: 'STRUCT'>, <DataType.UNION: 'UNION'>, <DataType.SET: 'SET'>, <DataType.GEOGRAPHY: 'GEOGRAPHY'>, <DataType.ENUM: 'ENUM'>, <DataType.JSON: 'JSON'>])
[2021-12-01 16:47:58,238] ERROR    {metadata.ingestion.source.glue:145} - None
[2021-12-01 16:47:58,238] ERROR    {metadata.ingestion.source.glue:146} - 1 validation error for Column
```

**Comment by harshach** [comment]
@matef Thanks for filing the issue. Assuming you did this run from main?

**Comment by ayush-shah** [comment]
Should be fixed with #1405.
Added support for the datatypes.

**Comment by harshach** [comment]
@ayush-shah can we create a table with map type and test it out
